corehq/apps/translations/utils.py

```diff
@@ -32,3 +32,33 @@ def update_app_translations_from_trans_dict(app, trans_dict):
         if isinstance(app, LinkedApplication):
             app.linked_app_translations.update(trans_dict)
         app.translations.update(trans_dict)
+
+
+def zip_with_gaps(all_items, some_items, allitem_keyfunc=None, someitem_keyfunc=None):
```

Comment:
Do you think I should move this to (the recently deleted) `corehq/util/itertools.py`?

Reply:
I have the opposite tendency: that is, not to put a function in a utils-type file until it's used in more than one place. But I think there's a reasonable case for either approach.